src/styles/styles.css

```diff
@@ -151,3 +151,20 @@ h2 {
 .address-card h2 {
   font-size: 1.15rem;
 }
+
+@keyframes spinner {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
You made a CSS loading spinner, that's cool. 

You can use the React loader library which has loads of loading animations in it. 

import * as Loader from 'react-loader-spinner' and then you can use Loader.WhateverTheLoaderIsCalled to choose specific load animations. Pretty cool for this kind of thing.